doc/man3/CMS_add0_cert.pod

```diff
@@ -20,7 +20,8 @@ CMS_add0_crl, CMS_add1_crl, CMS_get1_crls
 
 =head1 DESCRIPTION
 
-CMS_add0_cert() and CMS_add1_cert() add certificate I<cert> to I<cms>.
+CMS_add0_cert() and CMS_add1_cert() add certificate I<cert> to I<cms>
+unless it is already present.
```

Comment:
And documenting what does that really mean in terms of references.

Reply:
This was already part of the documentation, namely in the NOTES section.
Yet since it easily gets overlooked there, I've moved it (somewhat re-phrasing it) directly into the description of the two functions.